### ISSUE TITLE
ghpc v1.11 cannot read packer/custom-image module

### ISSUE THREAD

**Issue by Tristan-Kosciuch** [GoogleCloudPlatform/hpc-toolkit#861]
### Describe the bug

While trying to create a custom image with packer I encountered this error:

```
2023/01/26 12:04:06 Failed to read packer source directory modules/packer/custom-image
```

### Steps to reproduce

1. `ghpc create blueprints/create-dvmdostem-image.yaml --vars project_id=$PROJECTID --vars ssh_username=$USER`

### Expected behavior

A custom image will be created

### Actual behavior

ghpc could not find the source for the packer/custom-image module

### Version (`ghpc --version`)

```
ghpc version v1.11.0
Built from 'main' branch.
Commit info: v1.11.0-0-gd706498b
```

### Blueprint

If applicable, attach or paste the blueprint YAML used to produce the bug.

```yaml
blueprint_name: create-image

vars:
  project_id:  ## GCP Project ID ##
  ssh_username: ## Local username ##
  deployment_name: create-image
  region: us-central1
  zone: us-central1-c
  lustre_mgs_ip: 10.0.0.218@tcp
  startup_timeouts: 300
  network_name: slurm-gcp-v5-net
  subnetwork_name: slurm-gcp-v5-primary-subnet
  first_image_family: default-mpicc-dvmdostem-slurm
  second_image_family: default-mpicc-dvmdostem-lustre-slurm

deployment_groups:
- group: copy-image
  modules:
  - id: copy-image
    source: modules/packer/custom-image
    kind: packer
    settings:
      omit_external_ip: false
      disk_size: 32
      source_image_project_id: [schedmd-slurm-public]
      source_image_family: schedmd-v5-slurm-22-05-6-ubuntu-2004-lts
      image_family: schedmd-v5-slurm-22-05-6-ubuntu-2004-lts
```

### Execution environment

- OS: macOS
- Shell bash
- go version: go version go1.19.3 darwin/amd64

**Comment by tpdownes** [comment]
@Tristan-Kosciuch we believe that we resolved this issue in #827. This is present on the `develop` branch of the Toolkit and will be part of our next release. You can test whether this resolves your issue by running

```
git fetch --all
git checkout develop
```

before running `make` or `go build` to build the `ghpc` binary.

Please let us know if this addresses your issue.

**Comment by tpdownes** [comment]
@Tristan-Kosciuch we have released #827 in [v1.12.0](https://github.com/GoogleCloudPlatform/hpc-toolkit/releases/tag/v1.12.0). We believe this addresses your problem and will close the issue. Please re-open if you are still experiencing it.
